unattended.sh

```diff
@@ -1,186 +1,766 @@
+		fi
+	else
+		log "WARN" "Firewall was not detected"
+	fi
+}
+#========= end of function update_firewall_config()
 
+#========= begin of function enable_new_services()
+# enable newly added services to make them active after system reboot
 #
-# SERVICES
+function enable_new_services() {
+
+	log "INFO" "Enable and restart services ..."
+	if [ "x$has_systemd" '=' x1 ]; then
+		case $topology in
+
+		central)
+			log "DEBUG" "On central..."
+			systemctl enable mariadb $OS_SPEC_SERVICES snmpd snmptrapd gorgoned centreontrapd cbd centengine centreon
+			systemctl restart mariadb $OS_SPEC_SERVICES snmpd snmptrapd
+			;;
+
+		poller)
+			log "DEBUG" "On poller..."
+			systemctl enable centreon centengine centreontrapd snmptrapd
+			systemctl start centreontrapd snmptrapd
+			;;
+		esac
+	else
+		log "WARN" "Systemd not detected, skipping"
+	fi
+}
+#========= end of function enable_new_services()
+
+#========= begin of function setup_before_installation()
+# execute some tasks before installing Centreon
+# - disable SELinux
+# - install Centreon official repositories
+function setup_before_installation() {
+
+	set_runtime_selinux_mode "disabled"
+
+	install_centreon_repo
+}
+#========= end of function setup_before_installation()
+
+#========= begin of function install_wizard_post()
+# execute a post request of the install wizard
+# - session coocky
+# - php command
+# - request body
+function install_wizard_post() {
+	echo -n "wizard install step ${2} response -> "
+	curl -s "http://${central_ip}/centreon/install/steps/process/${2}" \
+		-H 'Content-Type: application/x-www-form-urlencoded; charset=UTF-8' \
+		-H "Cookie: ${1}" --data "${3}"
+}
+#========= end of function install_wizard_post()
+
+#========= begin of function play_install_wizard()
+function play_install_wizard() {
+	log "INFO" "Playing install wizard"
+
+	sessionID=$(curl -s -v "http://${central_ip}/centreon/install/install.php" 2>&1 | grep Set-Cookie | awk '{print $3}')
+	curl -s "http://${central_ip}/centreon/install/steps/step.php?action=stepContent" -H "Cookie: ${sessionID}" > /dev/null
+	install_wizard_post ${sessionID} "process_step3.php" 'install_dir_engine=%2Fusr%2Fshare%2Fcentreon-engine&centreon_engine_stats_binary=%2Fusr%2Fsbin%2Fcentenginestats&monitoring_var_lib=%2Fvar%2Flib%2Fcentreon-engine&centreon_engine_connectors=%2Fusr%2Flib64%2Fcentreon-connector&centreon_engine_lib=%2Fusr%2Flib64%2Fcentreon-engine&centreonplugins=%2Fusr%2Flib%2Fcentreon%2Fplugins%2F'
+	install_wizard_post ${sessionID} "process_step4.php" 'centreonbroker_etc=%2Fetc%2Fcentreon-broker&centreonbroker_cbmod=%2Fusr%2Flib64%2Fnagios%2Fcbmod.so&centreonbroker_log=%2Fvar%2Flog%2Fcentreon-broker&centreonbroker_varlib=%2Fvar%2Flib%2Fcentreon-broker&centreonbroker_lib=%2Fusr%2Fshare%2Fcentreon%2Flib%2Fcentreon-broker'
+	install_wizard_post ${sessionID} "process_step5.php" "admin_password=${centreon_admin_password}&confirm_password=${centreon_admin_password}&firstname=John&lastname=Doe&email=jd%40cie.tld"
```

Comment:
```suggestion
	install_wizard_post ${sessionID} "process_step5.php" "admin_password=${centreon_admin_password}&confirm_password=${centreon_admin_password}&firstname=${centreon_admin_firstname}&lastname=${centreon_admin_lastname}&email=${centreon_admin_email}"
```
Set first and last name of administrator